### DISCUSSION TITLE
Netherite-Tier Powertool(s)

### DISCUSSION THREAD

**Discussion by DaFuqs** [TechReborn/TechReborn#2523] [Ideas]
The Omnitool (and the other tools) is not able to mine blocks that can only be broken by netherite tool tiers.
I have to fall back to a netherite pickaxe currently, like some form of cave man.

**Comment by drcrazy** [comment]
Is it about modded blocks?
Vanilla diamond pickaxe can mine same blocks as netherite pickaxe: https://minecraft.gamepedia.com/Pickaxe

**Comment by DaFuqs** [comment]
Astromine (https://github.com/Mixinors/Astromine) adds a meteors that can only be broken by netherite tier tools. I can take a look on how that's implemented later today. Maybe adding Astromine messes something up - I can remember trying to mine some TR ores with the Onmitool and those dropping nothing, being shown as "non harvestable". Haven't found a suitable mixin on their side yet.

If you want to I can also create a small set of suggestions for better Astromine integration (or post it to astromine, if it's better suited there). For example both use power and iron plates that can be used interchangeable, but Astromine-exclusive materials can only be pressed by those machines currently.
Astromine also has fluid pipes that don't connect to TR machinery. Those would be dope for the industrial grinder. (both seem to use LibBlockAttributes though? EDIT: LBA was added to Astromine 10 days ago. I'll try out the dev build and see if it works)

**Comment by drcrazy** [comment]
List for Astromine compat will be good

**Comment by DaFuqs** [comment]
Since Omnitool was a separate fix from Astromine ores (and the TR version with the fix was approved on curse just now!) and I've created separate suggestion posts for astromine compat recipes most things here should be resolved.

Remaining:
- Astromines fluid cables still don't connect to TR machines, but that may well be on their end.
- Drills and Omnitool have harvest level 3, but Astromine utilises blocks that can only be broken with harvest level 4 (Netherite), but since vanilla goes only up to 3 its better for you to decide to add support or not. Especially because [their tools seem to go up to mining level 7](https://github.com/Mixinors/Astromine/blob/33c79f2398041a598132486b62f54653a66e1325/astromine-foundations/src/main/java/com/github/chainmailstudios/astromine/foundations/registry/AstromineFoundationsToolMaterials.java). There are only 2 blocks that need 5/6: The matching ingot blocks of Galaxium (5) and Univite (6).
